modules/RTC/TraceablePeerConnection.js

```diff
@@ -1797,6 +1806,71 @@ TraceablePeerConnection.prototype.setRemoteDescription = function(
 };
 
 /**
+ * Inserts an H264 payload into the description if not already present.
+ *
```

Comment:
Can you mention why this is needed here too? Hopefully we can get rid of it soon. A link to the issues would also help.